### ISSUE TITLE
Since new update aircon does not respond 

### ISSUE THREAD

**Issue by hauni97** [kiwi-cam/homebridge-broadlink-rm#226]
Aircon states no response in HK, without changing the config file in homebridge.

**Comment by kiwi-cam** [comment]
Could you provide more detail, I'm not seeing this issue with my aircon? What version are you running, and what do you see in the logs?

Also, providing your config.json can help to reproduce the issue.

**Comment by dnschnur** [comment]
I'm also seeing "No Response" for an aircon. I have 3 RMs connected, each with one AC. The "Living Room AC" is completely fine, while the other two show "No Response". However, Homekit **does** show temperature from the RM's temp sensor cable, and if I expand them I can switch modes; it transmits the IR signal.

So the problem is that, even though they are clearly active and connected, they show up in Homekit with the red "No Response" text, and the temperature wheel won't accept any input.

The configs for all three are identical:

```
                {
                    "name": "Bedroom AC",
                    "type": "air-conditioner",
                    "host": "10.0.0.13",
                    "data": {
                      ...
                    },
                    "defaultCoolTemperature": 20,
                    "defaultHeatTemperature": 20,
                    "minTemperature": 17,
                    "maxTemperature": 23,
                    "preventResendHex": false,
                    "temperatureDisplayUnits": "F",
                    "turnOnWhenOff": true
                },
                {
                    "name": "Living Room AC",
                    "type": "air-conditioner",
                    "host": "10.0.0.12",
                    "data": {
                      ...
                    },
                    "defaultCoolTemperature": 20,
                    "defaultHeatTemperature": 20,
                    "minTemperature": 17,
                    "maxTemperature": 23,
                    "preventResendHex": false,
                    "temperatureDisplayUnits": "F",
                    "turnOnWhenOff": true
                },
                {
                    "name": "Office AC",
                    "type": "air-conditioner",
                    "host": "10.0.0.11",
                    "data": {
                      ...
                    },
                    "defaultCoolTemperature": 20,
                    "defaultHeatTemperature": 20,
                    "minTemperature": 17,
                    "maxTemperature": 23,
                    "preventResendHex": false,
                    "temperatureDisplayUnits": "F",
                    "turnOnWhenOff": true
                },
```

**Comment by dnschnur** [comment]
I realized that my config was incorrect; the auto/head/cool values were hex-strings, rather than `{"data": <hex-string>}`.

That wasn't actually the problem, but after fixing it I turned on the AC again and restarted the Homebridge server, at which point the "No Response" went away. So the problem seems to be with the initial state of the device; once it's turned on and a temperature is set, the issue goes away.
